app/assets/javascripts/searchform.js

```diff
@@ -1,12 +0,0 @@
-  /**
```

Comment:
@jywarren I changed the way search worked a little. In my humble opinion using the keyword or query for search in route directly was causing all the search problem. Now it's in a different route `search/all` with a post form. I did delete the jquery file which was not necessary now.

Reply:
Hi, thank you! But i worry this will affect the ability to paste a URL of a given search -- like http://publiclab.org/search/sensors -- I think this means we're going to have to continue supporting the GET version with a route. Would it be substantially easier to do http://publiclab.org/search?q=sensors ?

Thanks, i know this is a tough one -- your creativity and work is appreciated!!!

Reply:
Hi, okay I will work on it. Was busy past 2 days couldn't reply sorry for that.